src/libraries/Native/Unix/System.Native/pal_console.c

```diff
@@ -19,6 +19,13 @@
 #include <pthread.h>
 #include <signal.h>
 
+#if HAVE_SIGACTION_ULONG_FLAGS
+// uClibc
+typedef unsigned long NativeFlagsType;
```

Comment:
Where is this unsigned long? the uClibs doc says that it should be unsigned long only on Sparc that we don't support (https://github.com/kraj/uClibc/blob/ca1c74d67dd115d059a875150e10b8560a9c35a8/docs/sigaction.txt#L2)

Reply:
https://github.com/kraj/uClibc/blob/266bdc1f623fe6fe489e5115e0f8ef723705d949/libc/sysdeps/linux/common/bits/sigaction.h#L39

Reply:
Ok, thanks for the details.